README.md

```diff
@@ -1,43 +1,65 @@
-cloud-sdk-docker
-================
 
-> :warning: **IMPORTANT NOTE:** The base image for the `cloud-sdk` image will be changed to Alpine Linux on or after 5/15/17.
-> For more information, please see: [#58](https://github.com/GoogleCloudPlatform/cloud-sdk-docker/issues/58)
----
+# Google Cloud SDK Docker
 
-[`google/cloud-sdk`](https://index.docker.io/u/google/cloud-sdk/) is a
-[Docker](https://docker.io) image bundling all the components and dependencies
-of the [Google Cloud SDK](https://cloud.google.com/sdk/) including alpha and
-beta components.
 
-## Usage
+Baseline image for the [Google Cloud SDK](https://cloud.google.com/sdk/).
 
-Follow these instructions if you are running docker *outside* of Google Compute Engine:
+Imaage includes default command line tools:
+*  gcloud:  Default set of gcloud commands
+*  gsutil:  Cloud Storage Command Line Tool
+*  bq: BigQuery Command Line Tool
 
-    # Get the cloud sdk image:
-    $ docker pull google/cloud-sdk
 
-    # Auth & save the credentials in gcloud-config volumes:
-    $ docker run -t -i --name gcloud-config google/cloud-sdk gcloud init
+## Dockerfile
```

Comment:
summarized the usage;
there's not supported tag reference to point to (it'll only exist after the first tag and push of the new image)  I added in the section though.

The intent of this git repo isn't exclusively to seed dockerhub - its also as a landing page that shows the various other dockerfiles.  eg. the debain based image is in this repo but is never build (we could later, though).  In that respect, i'd say the centOS dockerfile should get listed  back here too